Matveibos/hometask-6/my_hash_spec.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require_relative './my_hash'
+require 'pry'
+require 'rspec'
+
+describe MyHash do
+  subject(:my_hash) { described_class.new }
+
+  describe '#[]=' do
+    context 'when value does not exist' do
+      it 'return pair key-value' do
+        expect(my_hash[:key1] = '').to eq('')
```

Comment:
It's a bad practice to setup test variables inside of a test. Check let variables and how to use them in RSpec tests